pkg/gvk/compare.go

```diff
@@ -0,0 +1,280 @@
+package gvk
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/dynamic"
+
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+)
+
+var crdGVR = schema.GroupVersionResource{
```

Comment:
In case anyone else reviewing  is wondering: this is used for excluding all CRDs from the comparison list since CRDs can be moved over to a dest cluster already.